data/human/jobs.txt

```diff
@@ -2318,7 +2319,7 @@ mission "Strike Breakers [large mining]"
 	on visit
 		dialog phrase "generic passenger on visit"
 	on complete
-		payment 130000 100
+		payment 130000 120
```

Comment:
Is a big company that's trying to swindle their employees likely to pay people well to get rid of them?